collectors/misc/score/src/main/java/com/capitalone/dashboard/repository/ScoreApplicationRepository.java

```diff
@@ -0,0 +1,34 @@
+package com.capitalone.dashboard.repository;
+
+import java.util.List;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.mongodb.repository.Query;
+
+import com.capitalone.dashboard.model.ScoreApplication;
+import com.capitalone.dashboard.model.ScoreCollector;
+
+/**
+ * Repository for {@link ScoreApplication}s.
+ */
+public interface ScoreApplicationRepository extends BaseCollectorItemRepository<ScoreApplication> {
+
+  /**
+   * Find a {@link ScoreApplication} by dashboard id.
+   *
+   * @param collectorId ID of the {@link ScoreCollector}
+   * @param dashboardId DashboardId
+   * @return a {@link ScoreApplication} instance
+   */
+  @Query(value = "{ 'collectorId' : ?0, options.dashboardId : ?1}")
+  ScoreApplication findScoreByDashboard(ObjectId collectorId, ObjectId dashboardId);
```

Comment:
@sidbha007  - findScoreByDashboard() never used

Reply:
hv removed it